packages/core/src/index.ts

```diff
@@ -68,6 +68,7 @@ import * as GenerateDisposableToken from './messages/responses/generate-disposab
 // VectorClient Response Types
 export * as vector from './messages/responses/vector';
 export * from './messages/responses/vector';
+export {VectorIndexItem} from './messages/vector-index';
```

Comment:
In some of the `index.ts` files we first import a name, then later export it. I am just directly exporting it like so. I assume this is just a stylistic difference.